spec/unit/builders/builders.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+*/
+
+const rewire = require('rewire');
+
+const CordovaError = require('cordova-common').CordovaError;
+const ProjectBuilder = require('../../../bin/templates/cordova/lib/builders/ProjectBuilder');
+
+describe('builders', () => {
+    let builder;
+
+    beforeEach(() => {
+        builder = rewire('../../../bin/templates/cordova/lib/builders/builders');
+    });
+
+    describe('getBuilder', () => {
+        it('should return an instance of ProjectBuilder', () => {
+            const newBuilder = builder.getBuilder('gradle');
+            expect(newBuilder).toEqual(jasmine.any(ProjectBuilder));
+        });
+
+        it('should return an instance of ProjectBuilder', () => {
+            const newBuilder = builder.getBuilder('studio');
+            expect(newBuilder).toEqual(jasmine.any(ProjectBuilder));
+        });
```

Comment:
Duplicated test

Reply:
The first tests when `gradle` is requested, and the second when `studio` is requested. I will rename the tests to make the difference more obvious.